src/main/java/com/denizenscript/depenizen/bukkit/events/jobs/JobsJobsExpGainScriptEvent.java

```diff
@@ -72,6 +76,8 @@ public ObjectTag getContext(String name) {
             case "job" -> job;
             case "experience" -> new ElementTag(event.getExp());
             case "action" -> new ElementTag(event.getActionInfo().getType().getName(), true);
+            case "entity" -> new EntityTag(event.getLivingEntity());
```

Comment:
The docs say ` if applicable.` but this code always returns an entity - one of the two is wrong

Reply:
Yeah my bad. I've added a check for the entity being null 🙂